### DISCUSSION TITLE
How to work/enforce elemental hydrogen (H2) in Rdkit?

### DISCUSSION THREAD

**Discussion by DocMinus** [rdkit/rdkit#6305] [Q&A] [answered]
I have small app at work that looks at single components of a reaction and that may include hydrogen (gas). Unfortunately chem.molfromsmiles() only leads to None.
I tried as input "H", "HH" and "H-H".
As a result it cannot be displayed as PNG/SVG. I don't expect any property calculations to be done for hydrogen, but I would like to depict it without having to do (too many) if/then workarounds or drawing a manual image by non-Rdkit means. 
Is it possible?

**Comment by richardjgowers** [comment, marked as answer]
@DocMinus in SMILES you'll need to include square brackets around elements that aren't B, C, N, O, P, S, F, Cl, or Br, so `Chem.MolFromSmiles('[H][H]')`

**Reply by DocMinus** [reply]
doh!
so obvious in retrospec...
thanks!
